contributed_definitions/NXxpcs.nxdl.xml

```diff
@@ -523,8 +523,8 @@
             The only requirement for the list is that it may be iterable. Some expected formats are:
 
             * iterable list of floats (i.e., :math:`Q(r)`)
-            * iterable list of tuples (i.e., :math:`Q(r)`, :math:`\varphi`), but preferable use the seperate :math:`\varphi` field below
-            * iterable list of tuples (e.g., (H, K, L); (qx, qy, qz); (horizontal_pixel, vertical_pixel))
+            * iterable list of tuples (i.e., :math:`Q(r)`, :math:`\varphi`), but preferable use the separate :math:`\varphi` field below
```

Comment:
Which version is the intention here"
* :math:`Q(r)`, :math:`\varphi`
* :math:`Q(r, \varphi)`